chain/telemetry/README.md

```diff
@@ -0,0 +1,9 @@
+# Near Telemetry
+
+A small utility (TelemetryActor), that tries to send the telemetry (metrics) information as JSON over HTTP-post to selected list of servers.
```

Comment:
Could you explain the reason why we need this? Specifically: 

* what are those select servers?
* what they do? 

Perhaps there's some dashbord we can link which displays the data?

Reply:
I don't know what these servers are (see the TODO at the end of the README).

@frol @telezhnaya - do you happen to know where does this telemetry data end up ?

Reply:
[By default](https://github.com/near/nearcore/blob/1cbcc5eb89be9c78364d2ab984a192c77f9538aa/nearcore/src/config.rs#L849-L858), the telemetry [is configured](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json) to send the telemetry to NEAR Explorer.